sandbox/index.js

```diff
@@ -78,7 +78,7 @@ class Sandbox {
     this.options = options || {};
     this.provider = service.provider;
     this.snapshotId = undefined;
-    this.version = 'v0.5';
+    this.version = 'v0.6';
```

Comment:
it is odd to have `v` in version. I think we should avoid it:

```suggestion
    this.version = '0.6';
```